dracut/30ignition/ignition-diskful.target

```diff
@@ -7,6 +7,9 @@ Documentation=https://github.com/coreos/ignition
 ConditionPathExists=/etc/initrd-release
 Before=ignition-complete.target
 
+Requires=mnt-boot.mount
+After=mnt-boot.mount
```

Comment:
In general, I think it's cleaner to have the symbolic targets like this not pull in anything by themselves, but instead only pull stuff in by linking.

So here, we already statically link `mnt-boot.mount` to this unit, so we don't need `Requires=`. And instead of `After=` here, you'd do `Before=ignition-diskful.target` in `mnt-boot.mount`. (See for example how `ignition-complete.target` is wired up.)

Reply:
Updated.